client/src/containers/interventions/new/step1/component.tsx

```diff
@@ -168,7 +168,7 @@ const Step1: FC = () => {
             options={optionsBusinesses}
             placeholder="all businesses"
             theme="inline-primary"
-            onChange={({ value }) => handleDropdown('businessUnitsIds', value)}
+            onChange={(value) => handleDropdown('businessUnitsIds', value)}
```

Comment:
when you open the form for a new intervention the default option for this select should be "all business". Also, if you change the params for handleDropdown, you need to set the values differently in the function as we are saving just the business id (just the value of the option selected)

Reply:
You're right, that's already taken care of. Next time I'll check that all data is stored correctly, promise!